src/react-components/rtc-debug-panel.js

```diff
@@ -0,0 +1,1011 @@
+import React, { Component } from "react";
```

Comment:
I can see a few places where we can refactor the react into smaller components. Happy to work with you on this!